### ISSUE TITLE
train_from saved model with 8 GPUs have zombie process

### ISSUE THREAD

**Issue by ghost** [OpenNMT/OpenNMT-py#1854]
hi,  I've tried to train model from saved checkpoint.
But when GPU number is 8, there will be zombie process, and stop training at there

[2020-08-25 11:27:30,730 INFO] Starting training on GPU: [0, 1, 2, 3, 4, 5, 6, 7]
[2020-08-25 11:27:30,731 INFO] Start training loop and validate every 5000 steps...
[2020-08-25 11:27:35,346 INFO] number of examples: 999957

When I change GPU number to 4, It works fine.
OpenNMT-py version is 1.1.1
pytorch version is 1.4.0
torchtext version is 0.4.0
Is there any method to slove this problem?

**Comment by francoishernandez** [comment]
What's your command line?

**Comment by AbelCui** [comment]
> What's your command line?

train_steps: 1000
valid_steps: 5000
warmup_steps: 8000
report_every: 100

model_dtype: fp16
decoder_type: transformer
encoder_type: transformer
self_attn_type: average
enc_layers: 12
dec_layers: 6
rnn_size: 1024
word_vec_size: 1024
transformer_ff: 4096
heads: 16

accum_count: 4
optim: adam
adam_beta1: 0.9
adam_beta2: 0.98
decay_method: noam
learning_rate: 2
learning_rate_decay: 1.0
max_grad_norm: 0.0

batch_size: 2048
batch_type: tokens
normalization: tokens
dropout: 0.1
attention_dropout: 0.1
label_smoothing: 0.1

max_generator_batches: 2

param_init: 0.0
param_init_glorot: 'true'
position_encoding: 'true'

world_size: 8
gpu_ranks:
- 0
- 1
- 2
- 3
- 4
- 5
- 6
- 7

**Comment by AbelCui** [comment]
> What's your command line?

BTW, I use train_from to reload saved model, the saved model is trained with same parameters but no train_from.
